notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -20,6 +20,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        n_obs = df.query(df_query).groupby("victim_visits").adv_win.count()
</pre><p>Why create the groupby object twice rather than reusing the one from previous step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Fixed to use a single groupby, mirroring the structure in later cells.</p><p> </p>